### ISSUE TITLE
Topgrade not updating from AUR

### ISSUE THREAD

**Issue by Locutus64** [topgrade-rs/topgrade#468]
## Additional Details
- XerpLinux - Arch based
- Topgrade was insstalled from the Konsole using Pacseek
- Topgrade 11.0.2-1

~~~
░▒▓ ~  topgrade --dry-run
Due to a design issue with notify-send it could be that topgrade hangs when it's finished.
If this is the case on your system add the --skip-notify flag to the topgrade command or set skip_notify = true in the config file.
If you don't want this message to appear any longer set display_preamble = false in the config file.
For more information about this issue see https://askubuntu.com/questions/110969/notify-send-ignores-timeout and https://github.com/topgrade-rs/topgrade/issues/288.

── 05:11:41 - System update ────────────────────────────────────────────────────
Dry running: /usr/bin/sudo pacman -Syu --noconfirm

── 05:11:41 - oh-my-zsh ────────────────────────────────────────────────────────
Pulling custom plugins and themes
Would pull /home/locutus/.oh-my-zsh/custom/plugins/zsh-completions
Would pull /home/locutus/.oh-my-zsh/custom/plugins/zsh-syntax-highlighting
Would pull /home/locutus/.oh-my-zsh/custom/plugins/zsh-autosuggestions
Would pull /home/locutus/.oh-my-zsh/custom/themes/powerlevel10k
Dry running: zsh /home/locutus/.oh-my-zsh/tools/upgrade.sh

── 05:11:41 - TLDR ─────────────────────────────────────────────────────────────
Dry running: /usr/bin/tldr --update
Dry running: /usr/bin/dotnet tool list --global

── 05:11:41 - Node Package Manager ─────────────────────────────────────────────

── 05:11:42 - Flatpak User Packages ────────────────────────────────────────────
Dry running: /usr/bin/flatpak update --user -y

── 05:11:42 - Flatpak System Packages ──────────────────────────────────────────
Dry running: /usr/bin/flatpak update --system -y

── 05:11:42 - snap ─────────────────────────────────────────────────────────────
Dry running: /usr/bin/sudo /usr/bin/snap refresh

── 05:11:42 - Firmware upgrades ────────────────────────────────────────────────
Dry running: /usr/bin/fwupdmgr refresh
Dry running: /usr/bin/fwupdmgr update -y

── 05:11:42 - Summary ──────────────────────────────────────────────────────────
System update: OK
oh-my-zsh: OK
TLDR: OK
Flatpak: OK
snap: OK
Firmware upgrades: OK
~~~

**Comment by SteveLauC** [comment]
Hi:)

Would you like to give me the output of `topgrade --dry-run --verbose` so that I can tell whether `pikaur` was invoked or not?

And have you set this config option to specify the package manager to use?

https://github.com/topgrade-rs/topgrade/blob/a7ddf4575a7f2962f87194299b3ff10576c823b0/config.example.toml#L90-L92

**Comment by Locutus64** [comment]
~~~
░▒▓ ~  topgrade --dry-run --verbose
DEBUG Configuration at /home/locutus/.config/topgrade.toml
DEBUG Loaded configuration: ConfigFile { sudo_command: None, pre_sudo: None, pre_commands: Some({}), post_commands: None, commands: Some({}), git_repos: None, predefined_git_repos: None, disable: Some([Emacs, Nix]), ignore_failures: None, remote_topgrades: None, remote_topgrade_path: None, ssh_arguments: None, git_arguments: None, tmux_arguments: None, set_title: None, display_time: None, display_preamble: None, assume_yes: Some(true), yay_arguments: None, aura_aur_arguments: None, aura_pacman_arguments: None, python: None, no_retry: None, run_in_tmux: None, cleanup: None, notify_each_step: None, accept_all_windows_updates: None, skip_notify: None, bashit_branch: None, only: None, composer: Some(Composer { self_update: None }), brew: Some(Brew { greedy_cask: None, autoremove: None }), linux: Some(Linux { yay_arguments: Some("--nodevel"), aura_aur_arguments: None, aura_pacman_arguments: None, arch_package_manager: Some(Pacman), show_arch_news: Some(true), garuda_update_arguments: None, trizen_arguments: None, pikaur_arguments: None, pamac_arguments: None, dnf_arguments: None, nix_arguments: None, apt_arguments: None, enable_tlmgr: None, redhat_distro_sync: None, rpm_ostree: None, emerge_sync_flags: None, emerge_update_flags: None }), git: Some(Git { max_concurrency: None, arguments: None, repos: None, pull_predefined: None }), windows: Some(Windows { accept_all_updates: None, self_rename: None, open_remotes_in_new_terminal: None, enable_winget: None, wsl_update_pre_release: None, wsl_update_use_web_download: None }), npm: Some(NPM { use_sudo: None }), yarn: None, vim: None, firmware: Some(Firmware { upgrade: Some(true) }), vagrant: None, flatpak: Some(Flatpak { use_sudo: None }), distrobox: None, no_self_update: None }
DEBUG Detected "/usr/bin/notify-send" as "notify-send"
DEBUG Version: 11.0.2
DEBUG OS: x86_64-unknown-linux-gnu
DEBUG Args { inner: ["topgrade", "--dry-run", "--verbose"] }
DEBUG Binary path: Ok("/usr/bin/topgrade")
DEBUG Self Update: false
Due to a design issue with notify-send it could be that topgrade hangs when it's finished.
If this is the case on your system add the --skip-notify flag to the topgrade command or set skip_notify = true in the config file.
If you don't want this message to appear any longer set display_preamble = false in the config file.
For more information about this issue see https://askubuntu.com/questions/110969/notify-send-ignores-timeout and https://github.com/topgrade-rs/topgrade/issues/288.
DEBUG Detected "/usr/bin/git" as "git"
DEBUG Cannot find "doas"
DEBUG Cannot find "please"
DEBUG Detected "/usr/bin/sudo" as "sudo"
DEBUG Cannot find "pwsh"
DEBUG Cannot find "powershell"
DEBUG Step "System update"

── 11:36:57 - System update ────────────────────────────────────────────────────
DEBUG Cannot find "powerpill"
DEBUG Cannot find "powerpill"
Dry running: /usr/bin/sudo pacman -Syu --noconfirm
DEBUG Step "config-update"
DEBUG Step "Brew"
DEBUG Step "yadm"
DEBUG Step "guix"
DEBUG Step "home-manager"
DEBUG Step "asdf"
DEBUG Step "pkgin"
DEBUG Step "bun"
DEBUG Path "/home/locutus/.config/emacs" doesn't exist
DEBUG Path "/home/locutus/.emacs.d" doesn't exist
DEBUG /home/locutus/.vim does not exists
DEBUG /home/locutus/.config/nvim does not exists
DEBUG /home/locutus/.ideavimrc does not exists
DEBUG /home/locutus/.intellimacs does not exists
DEBUG /home/locutus/.dotfiles does not exists
DEBUG /home/locutus/.zshrc is a file. Checking /home/locutus
DEBUG Checking if /home/locutus is a git repository
DEBUG Executing command `/usr/bin/git rev-parse --show-toplevel`
DEBUG Command failed: Err(
   0: Command failed: `/usr/bin/git rev-parse --show-toplevel`

      Stderr:
      fatal: not a git repository (or any parent up to mount point /)
      Stopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).
   1: `/usr/bin/git` failed: exit status: 128

Location:
   src/steps/git.rs:167

Backtrace omitted. Run with RUST_BACKTRACE=1 environment variable to display it.
Run with RUST_BACKTRACE=full to include source snippets.)
DEBUG /home/locutus/.tmux does not exists
DEBUG /home/locutus/.config/fish does not exists
DEBUG /home/locutus/.config/openbox does not exists
DEBUG /home/locutus/.config/bspwm does not exists
DEBUG /home/locutus/.config/i3 does not exists
DEBUG /home/locutus/.config/sway does not exists
DEBUG Step "Git repositories"
DEBUG Step "zr"
DEBUG Detected "/usr/bin/zsh" as "zsh"
DEBUG Step "antibody"
DEBUG Detected "/usr/bin/zsh" as "zsh"
DEBUG Step "antidote"
DEBUG Detected "/usr/bin/zsh" as "zsh"
DEBUG Step "antigen"
DEBUG Detected "/usr/bin/zsh" as "zsh"
DEBUG Path "/home/locutus/.zshrc" exists
DEBUG Step "zgenom"
DEBUG Detected "/usr/bin/zsh" as "zsh"
DEBUG Path "/home/locutus/.zshrc" exists
DEBUG Step "zplug"
DEBUG Detected "/usr/bin/zsh" as "zsh"
DEBUG Path "/home/locutus/.zshrc" exists
DEBUG Step "zinit"
DEBUG Detected "/usr/bin/zsh" as "zsh"
DEBUG Path "/home/locutus/.zshrc" exists
DEBUG Step "zi"
DEBUG Detected "/usr/bin/zsh" as "zsh"
DEBUG Path "/home/locutus/.zshrc" exists
DEBUG Step "zim"
DEBUG Detected "/usr/bin/zsh" as "zsh"
DEBUG Executing command `zsh -c '[[ -n ${ZIM_HOME} ]] && print -n ${ZIM_HOME}'`
DEBUG Command failed: Err(
   0: Command failed: `zsh -c '[[ -n ${ZIM_HOME} ]] && print -n ${ZIM_HOME}'`
   1: `zsh` failed: exit status: 1

Location:
   src/steps/zsh.rs:140

Backtrace omitted. Run with RUST_BACKTRACE=1 environment variable to display it.
Run with RUST_BACKTRACE=full to include source snippets.)
DEBUG Step "oh-my-zsh"
DEBUG Detected "/usr/bin/zsh" as "zsh"
DEBUG Path "/home/locutus/.oh-my-zsh" exists

── 11:36:57 - oh-my-zsh ────────────────────────────────────────────────────────
DEBUG Executing command `zsh -c 'test $ZSH_CUSTOM && echo -n $ZSH_CUSTOM'`
DEBUG Command failed: Err(
   0: Command failed: `zsh -c 'test $ZSH_CUSTOM && echo -n $ZSH_CUSTOM'`
   1: `zsh` failed: exit status: 1

Location:
   src/steps/zsh.rs:166

Backtrace omitted. Run with RUST_BACKTRACE=1 environment variable to display it.
Run with RUST_BACKTRACE=full to include source snippets.)
DEBUG Running zsh returned Command failed: `zsh -c 'test $ZSH_CUSTOM && echo -n $ZSH_CUSTOM'`. Using default path: /home/locutus/.oh-my-zsh/custom
DEBUG oh-my-zsh custom dir: /home/locutus/.oh-my-zsh/custom
DEBUG Checking if /home/locutus/.oh-my-zsh/custom is a git repository
DEBUG Executing command `/usr/bin/git rev-parse --show-toplevel`
DEBUG Checking if /home/locutus/.oh-my-zsh/custom/plugins is a git repository
DEBUG Executing command `/usr/bin/git rev-parse --show-toplevel`
DEBUG Checking if /home/locutus/.oh-my-zsh/custom/plugins/example is a git repository
DEBUG Executing command `/usr/bin/git rev-parse --show-toplevel`
DEBUG Checking if /home/locutus/.oh-my-zsh/custom/plugins/zsh-completions is a git repository
DEBUG Executing command `/usr/bin/git rev-parse --show-toplevel`
DEBUG Checking if /home/locutus/.oh-my-zsh/custom/plugins/zsh-autosuggestions is a git repository
DEBUG Executing command `/usr/bin/git rev-parse --show-toplevel`
DEBUG Checking if /home/locutus/.oh-my-zsh/custom/plugins/zsh-syntax-highlighting is a git repository
DEBUG Executing command `/usr/bin/git rev-parse --show-toplevel`
DEBUG Checking if /home/locutus/.oh-my-zsh/custom/themes is a git repository
DEBUG Executing command `/usr/bin/git rev-parse --show-toplevel`
DEBUG Checking if /home/locutus/.oh-my-zsh/custom/themes/powerlevel10k is a git repository
DEBUG Executing command `/usr/bin/git rev-parse --show-toplevel`
DEBUG /home/locutus/.oh-my-zsh/custom/themes/example.zsh-theme is a file. Checking /home/locutus/.oh-my-zsh/custom/themes
DEBUG Checking if /home/locutus/.oh-my-zsh/custom/themes is a git repository
DEBUG Executing command `/usr/bin/git rev-parse --show-toplevel`
DEBUG /home/locutus/.oh-my-zsh/custom/example.zsh is a file. Checking /home/locutus/.oh-my-zsh/custom
DEBUG Checking if /home/locutus/.oh-my-zsh/custom is a git repository
DEBUG Executing command `/usr/bin/git rev-parse --show-toplevel`
Pulling custom plugins and themes
Would pull /home/locutus/.oh-my-zsh/custom/plugins/zsh-autosuggestions
Would pull /home/locutus/.oh-my-zsh/custom/plugins/zsh-completions
Would pull /home/locutus/.oh-my-zsh/custom/plugins/zsh-syntax-highlighting
Would pull /home/locutus/.oh-my-zsh/custom/themes/powerlevel10k
Dry running: zsh /home/locutus/.oh-my-zsh/tools/upgrade.sh
DEBUG Step "fisher"
DEBUG Step "bash-it"
DEBUG Step "oh-my-fish"
DEBUG Step "fish-plug"
DEBUG Step "fundle"
DEBUG Step "tmux"
DEBUG Step "TLDR"
DEBUG Detected "/usr/bin/tldr" as "tldr"

── 11:36:57 - TLDR ─────────────────────────────────────────────────────────────
Dry running: /usr/bin/tldr --update
DEBUG Step "pearl"
DEBUG Step "Gnome Shell Extensions"
DEBUG Detected "/usr/bin/gdbus" as "gdbus"
DEBUG Step "SDKMAN!"
DEBUG Detected "/usr/bin/bash" as "bash"
DEBUG Step "rcm"
DEBUG Step "apm"
DEBUG Step "fossil"
DEBUG Step "rustup"
DEBUG Step "juliaup"
DEBUG Step ".NET"
DEBUG Detected "/usr/bin/dotnet" as "dotnet"
Dry running: /usr/bin/dotnet tool list --global
DEBUG Step "choosenim"
DEBUG Step "cargo"
DEBUG Step "Flutter"
DEBUG Step "go-global-update"
DEBUG Detected "/usr/bin/go" as "go"
DEBUG Executing command `/usr/bin/go env GOPATH`
DEBUG Step "gup"
DEBUG Detected "/usr/bin/go" as "go"
DEBUG Executing command `/usr/bin/go env GOPATH`
DEBUG Step "opam"
DEBUG Step "vcpkg"
DEBUG Step "pipx"
DEBUG Step "conda"
DEBUG Step "mamba"
DEBUG Step "pip3"
DEBUG Detected "/usr/bin/python3" as "python3"
DEBUG Executing command `/usr/bin/python3 -m pip`
DEBUG Command failed: Err(
   0: Command failed: `/usr/bin/python3 -m pip`

      Stderr:
      /usr/bin/python3: No module named pip
   1: `/usr/bin/python3` failed: exit status: 1

Location:
   src/steps/generic.rs:369

Backtrace omitted. Run with RUST_BACKTRACE=1 environment variable to display it.
Run with RUST_BACKTRACE=full to include source snippets.)
DEBUG Step "pip-review"
DEBUG Step "pipupgrade"
DEBUG Step "ghcup"
DEBUG Step "stack"
DEBUG Step "tlmgr"
DEBUG Step "myrepos"
DEBUG Step "chezmoi"
DEBUG Step "jetpack"
DEBUG Step "vim"
DEBUG Step "Neovim"
DEBUG Step "The Ultimate vimrc"
DEBUG Step "voom"
DEBUG Step "Kakoune"
DEBUG Step "helix"
DEBUG Step "npm"
DEBUG Detected "/usr/bin/npm" as "npm"

── 11:36:57 - Node Package Manager ─────────────────────────────────────────────
DEBUG Executing command `/usr/bin/npm --version`
DEBUG Executing command `/usr/bin/npm root '--location=global'`
DEBUG Step "yarn"
DEBUG Step "pnpm"
DEBUG Step "Containers"
DEBUG Step "deno"
DEBUG Step "composer"
DEBUG Step "krew"
DEBUG Step "helm"
DEBUG Step "gem"
DEBUG Step "rubygems"
DEBUG Step "julia"
DEBUG Step "haxelib"
DEBUG Step "sheldon"
DEBUG Step "rtcl"
DEBUG Step "bin"
DEBUG Step "gcloud"
DEBUG Step "micro"
DEBUG Step "raco"
DEBUG Step "spicetify"
DEBUG Step "GitHub CLI Extensions"
DEBUG Step "am"
DEBUG Step "deb-get"
DEBUG Step "toolbx"
DEBUG Step "Flatpak"
DEBUG Detected "/usr/bin/flatpak" as "flatpak"

── 11:36:58 - Flatpak User Packages ────────────────────────────────────────────
Dry running: /usr/bin/flatpak update --user -y

── 11:36:58 - Flatpak System Packages ──────────────────────────────────────────
Dry running: /usr/bin/flatpak update --system -y
DEBUG Step "snap"
DEBUG Detected "/usr/bin/snap" as "snap"

── 11:36:58 - snap ─────────────────────────────────────────────────────────────
Dry running: /usr/bin/sudo /usr/bin/snap refresh
DEBUG Step "pacstall"
DEBUG Step "pacdef"
DEBUG Step "protonup"
DEBUG Step "distrobox"
DEBUG Step "dkp-pacman"
DEBUG Step "pihole"
DEBUG Step "Firmware upgrades"
DEBUG Detected "/usr/bin/fwupdmgr" as "fwupdmgr"
DEBUG Executing command `uname -r`
DEBUG Uname output: 6.3.6-arch1-1


── 11:36:58 - Firmware upgrades ────────────────────────────────────────────────
Dry running: /usr/bin/fwupdmgr refresh
Dry running: /usr/bin/fwupdmgr update -y
DEBUG Step "Restarts"
DEBUG Step "Vagrant boxes"

── 11:36:58 - Summary ──────────────────────────────────────────────────────────
System update: OK
config-update: SKIPPED: Skipped in --yes
Brew: SKIPPED: Cannot find "brew" in PATH
yadm: SKIPPED: Cannot find "yadm" in PATH
guix: SKIPPED: Cannot find "guix" in PATH
home-manager: SKIPPED: Cannot find "home-manager" in PATH
asdf: SKIPPED: Cannot find "asdf" in PATH
pkgin: SKIPPED: Cannot find "pkgin" in PATH
bun: SKIPPED: Cannot find "bun" in PATH
Git repositories: SKIPPED: No repositories to pull
zr: SKIPPED: Cannot find "zr" in PATH
antibody: SKIPPED: Cannot find "antibody" in PATH
antidote: SKIPPED: Path "/home/locutus/.antidote" doesn't exist
antigen: SKIPPED: Path "/home/locutus/antigen.zsh" doesn't exist
zgenom: SKIPPED: Path "/home/locutus/.zgenom" doesn't exist
zplug: SKIPPED: Path "/home/locutus/.zplug" doesn't exist
zinit: SKIPPED: Path "/home/locutus/.zinit" doesn't exist
zi: SKIPPED: Path "/home/locutus/.zi" doesn't exist
zim: SKIPPED: Path "/home/locutus/.zim" doesn't exist
oh-my-zsh: OK
fisher: SKIPPED: Cannot find "fish" in PATH
bash-it: SKIPPED: Path "/home/locutus/.bash_it" doesn't exist
oh-my-fish: SKIPPED: Cannot find "fish" in PATH
fish-plug: SKIPPED: Cannot find "fish" in PATH
fundle: SKIPPED: Cannot find "fish" in PATH
tmux: SKIPPED: Path "/home/locutus/.tmux/plugins/tpm/bin/update_plugins" doesn't exist
TLDR: OK
pearl: SKIPPED: Cannot find "pearl" in PATH
Gnome Shell Extensions: SKIPPED: Desktop doest not appear to be gnome
SDKMAN!: SKIPPED: Path "/home/locutus/.sdkman/bin/sdkman-init.sh" doesn't exist
rcm: SKIPPED: Cannot find "rcup" in PATH
apm: SKIPPED: Cannot find "apm" in PATH
fossil: SKIPPED: Cannot find "fossil" in PATH
rustup: SKIPPED: Cannot find "rustup" in PATH
juliaup: SKIPPED: Cannot find "juliaup" in PATH
.NET: SKIPPED: Error running `dotnet tool list`. This is expected when a dotnet runtime is installed but no SDK.
choosenim: SKIPPED: Cannot find "choosenim" in PATH
cargo: SKIPPED: Path "/home/locutus/.cargo" doesn't exist
Flutter: SKIPPED: Cannot find "flutter" in PATH
go-global-update: SKIPPED: Path "/home/locutus/go/bin/go-global-update" doesn't exist
gup: SKIPPED: Path "/home/locutus/go/bin/gup" doesn't exist
opam: SKIPPED: Cannot find "opam" in PATH
vcpkg: SKIPPED: Cannot find "vcpkg" in PATH
pipx: SKIPPED: Cannot find "pipx" in PATH
conda: SKIPPED: Cannot find "conda" in PATH
mamba: SKIPPED: Cannot find "mamba" in PATH
pip3: SKIPPED: pip does not exists
pip-review: SKIPPED: Cannot find "pip-review" in PATH
pipupgrade: SKIPPED: Cannot find "pipupgrade" in PATH
ghcup: SKIPPED: Cannot find "ghcup" in PATH
stack: SKIPPED: Cannot find "stack" in PATH
tlmgr: SKIPPED: tlmgr must be explicity enabled in the configuration to run in Android/Linux
myrepos: SKIPPED: Cannot find "mr" in PATH
chezmoi: SKIPPED: Cannot find "chezmoi" in PATH
jetpack: SKIPPED: Cannot find "jetpack" in PATH
vim: SKIPPED: Cannot find "vim" in PATH
Neovim: SKIPPED: Cannot find "nvim" in PATH
The Ultimate vimrc: SKIPPED: Path "/home/locutus/.vim_runtime" doesn't exist
voom: SKIPPED: Cannot find "voom" in PATH
Kakoune: SKIPPED: Cannot find "kak" in PATH
helix: SKIPPED: Cannot find "helix" in PATH
npm: SKIPPED: NPM root is owned by another user which is not the current user. Set use_sudo = true under the NPM section in your configuration to run NPM as sudo
yarn: SKIPPED: Cannot find "yarn" in PATH
pnpm: SKIPPED: Cannot find "pnpm" in PATH
Containers: SKIPPED: Cannot find "docker" in PATH
deno: SKIPPED: Cannot find "deno" in PATH
composer: SKIPPED: Cannot find "composer" in PATH
krew: SKIPPED: Cannot find "kubectl-krew" in PATH
helm: SKIPPED: Cannot find "helm" in PATH
gem: SKIPPED: Cannot find "gem" in PATH
rubygems: SKIPPED: Path "/home/locutus/.gem" doesn't exist
julia: SKIPPED: Cannot find "julia" in PATH
haxelib: SKIPPED: Cannot find "haxelib" in PATH
sheldon: SKIPPED: Cannot find "sheldon" in PATH
rtcl: SKIPPED: Cannot find "rupdate" in PATH
bin: SKIPPED: Cannot find "bin" in PATH
gcloud: SKIPPED: Cannot find "gcloud" in PATH
micro: SKIPPED: Cannot find "micro" in PATH
raco: SKIPPED: Cannot find "raco" in PATH
spicetify: SKIPPED: Cannot find "spicetify" in PATH
GitHub CLI Extensions: SKIPPED: Cannot find "gh" in PATH
am: SKIPPED: Cannot find "am" in PATH
deb-get: SKIPPED: Cannot find "deb-get" in PATH
toolbx: SKIPPED: Cannot find "toolbox" in PATH
Flatpak: OK
snap: OK
pacstall: SKIPPED: Cannot find "pacstall" in PATH
pacdef: SKIPPED: Cannot find "pacdef" in PATH
protonup: SKIPPED: Cannot find "protonup" in PATH
distrobox: SKIPPED: Cannot find "distrobox" in PATH
dkp-pacman: SKIPPED: Cannot find "dkp-pacman" in PATH
pihole: SKIPPED: Cannot find "pihole" in PATH
Firmware upgrades: OK
Restarts: SKIPPED: Cannot find "needrestart" in PATH
Vagrant boxes: SKIPPED: Cannot find "vagrant" in PATH

Pacman backup configuration files found:
/etc/conf.d/snapper.pacsave
/etc/makepkg.conf.pacnew
/etc/pacman.conf.pacnew
DEBUG Desktop notification: Topgrade finished successfully
DEBUG Executing command `/usr/bin/notify-send -t 10000 -a Topgrade Topgrade 'Topgrade finished successfully'`
~~~

~~~
[linux]
# Arch Package Manager to use. Allowed values: autodetect, trizen, paru, yay, pikaur, pacman.
~~~

**Comment by SteveLauC** [comment]
It seems that you have set:

```toml
[linux]
arch_package_manager = pacman
```

> Your configuration:
>
> DEBUG Loaded configuration: ConfigFile { sudo_command: None, pre_sudo: None, pre_commands: Some({}), post_commands: None, commands: Some({}), git_repos: None, predefined_git_repos: None, disable: Some([Emacs, Nix]), ignore_failures: None, remote_topgrades: None, remote_topgrade_path: None, ssh_arguments: None, git_arguments: None, tmux_arguments: None, set_title: None, display_time: None, display_preamble: None, assume_yes: Some(true), yay_arguments: None, aura_aur_arguments: None, aura_pacman_arguments: None, python: None, no_retry: None, run_in_tmux: None, cleanup: None, notify_each_step: None, accept_all_windows_updates: None, skip_notify: None, bashit_branch: None, only: None, composer: Some(Composer { self_update: None }), brew: Some(Brew { greedy_cask: None, autoremove: None }), linux: Some(Linux { yay_arguments: Some("--nodevel"), aura_aur_arguments: None, aura_pacman_arguments: None, * **arch_package_manager: Some(Pacman),** * show_arch_news: Some(true), garuda_update_arguments: None, trizen_arguments: None, pikaur_arguments: None, pamac_arguments: None, dnf_arguments: None, nix_arguments: None, apt_arguments: None, enable_tlmgr: None, redhat_distro_sync: None, rpm_ostree: None, emerge_sync_flags: None, emerge_update_flags: None }), git: Some(Git { max_concurrency: None, arguments: None, repos: None, pull_predefined: None }), windows: Some(Windows { accept_all_updates: None, self_rename: None, open_remotes_in_new_terminal: None, enable_winget: None, wsl_update_pre_release: None, wsl_update_use_web_download: None }), npm: Some(NPM { use_sudo: None }), yarn: None, vim: None, firmware: Some(Firmware { upgrade: Some(true) }), vagrant: None, flatpak: Some(Flatpak { use_sudo: None }), distrobox: None, no_self_update: None }


to ask `topgrade` to use `pacman` instead of `pikaur`

-----

To solve the issue:
1. Remove that configuration option and `topgrade` will use `pikaur` since it is preferred in our implementation
2. Explicit set `arch_package_manager = "pikaur` to ask `topgrade` to use it.

**Comment by Locutus64** [comment]
Hows that happen since all I do is install pikaur, pacseek, and topgrade in that order and the only configuration file I change is for pacseek to use pikaur instead of yay?

**Comment by SteveLauC** [comment]
So have you set that option? 

Or there perhaps is something wrong with the configuration loading?

**Comment by Locutus64** [comment]
No I diudn't set anything like I said, perhaps just a one off. I mean it's the first time I've run across the issus. It's working now, so thanks a lot.
